### ISSUE TITLE
Installation of ZM Engine fails in CentOS 7

### ISSUE THREAD

**Issue by sandoche2k** [zonemaster/zonemaster-engine#830]
```sh
[...]

Remember to make sure all of the above names are in the
MANIFEST file, or they will not be installed.

make[1]: Leaving directory `/root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/share'
PERL_DL_NONLAZY=1 /usr/bin/perl "-MExtUtils::Command::MM" "-e" "test_harness(0, 'inc', 'blib/lib', 'blib/arch')" t/*.t
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
# Testing Zonemaster Engine v3.1.2, Perl 5.016003, /usr/bin/perl
t/00-load.t ............... ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
Use of uninitialized value $asn4 in array dereference at t/asn.t line 28.
Use of uninitialized value in numeric eq (==) at t/asn.t line 28.
t/asn.t ................... ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/dnsname.t ............... ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Constants.pm line 13.
t/logger.t ................ ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/nameserver-axfr.t ....... ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/nameserver-axfr.t ....... ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/nameserver.t ............ ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/old-bugs.t .............. ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/pod-coverage.t .......... ok
t/pod.t ................... ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Constants.pm line 13.
t/profiles.t .............. ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/recursor.t .............. ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at t/Test-address.t line 4.
t/Test-address.t .......... ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/Test-basic.t ............ ok
t/Test-connectivity.t ..... ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/Test-consistency.t ...... ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/Test-consistency05-A.t .. ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/Test-consistency05-E.t .. ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/Test-consistency05-F.t .. ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/Test-consistency05-G.t .. ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/Test-consistency05-H.t .. ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/Test-consistency05-I.t .. ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/Test-consistency05-J.t .. ok
...
...
t/Test-delegation03-B.t ... ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/Test-delegation03-C.t ... ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/Test-dnssec-more.t ...... ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.

#   Failed test 'afnic.fr gives ALL_ALGO_SIGNED'
#   at t/Test-dnssec.t line 25.

#   Failed test 'afnic.fr does not give RRSIG_BROKEN'
#   at t/Test-dnssec.t line 36.
# Looks like you failed 2 tests of 270.
t/Test-dnssec.t ...........
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/270 subtests
        (less 2 skipped subtests: 266 okay)
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/Test-dnssec05-A.t ....... ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/Test-dnssec05-B.t ....... ok

...
...
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/undelegated.t ........... ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Recursor.pm line 11.
t/util.t .................. ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/zone.t .................. ok
Subroutine Zonemaster::Engine::Net::IP::Error redefined at /root/.cpanm/work/1604179492.24168/Zonemaster-Engine-v3.1.2/blib/lib/Zonemaster/Engine/Profile.pm line 18.
t/zonemaster.t ............ ok

Test Summary Report
-------------------
t/old-bugs.t            (Wstat: 0 Tests: 16 Failed: 0)
  TODO passed:   4
t/Test-dnssec.t         (Wstat: 512 Tests: 270 Failed: 2)
  Failed tests:  244, 249
  Non-zero exit status: 2
Files=84, Tests=1551, 586 wallclock secs ( 0.40 usr  0.11 sys + 573.92 cusr  8.15 csys = 582.58 CPU)
Result: FAIL
Failed 1/84 test programs. 2/1551 subtests failed.
make: *** [test_dynamic] Error 255
-> FAIL Installing Zonemaster-Engine-v3.1.2.tar.gz failed. See /root/.cpanm/work/1604179492.24168/build.log for details. Retry with --force to force install it.
1 distribution installed


```

**Comment by matsduf** [comment]
@sandoche2k, is this fixed by #831? Can this issue be closed?

**Comment by sandoche2k** [comment]
https://github.com/zonemaster/zonemaster-engine/pull/831
